go/tao/rollback.go

```diff
@@ -79,11 +72,187 @@ func Unprotect(keys []byte, in []byte) ([]byte, error) {
 	hm.Write(in[32:])
 	calculatedHmac := hm.Sum(nil)
 	if bytes.Compare(calculatedHmac, in[0:32]) != 0 {
-		return nil, errors.New("Unprotect: Bad mac")
+		return nil, errors.New("unprotect: Bad mac")
 	}
 	return out, nil
 }
 
+// Initialize rollback data.
+func InitRollbackState(tableName string, sealsBeforeSave int) {
```

Comment:
Not sure I see what this function does...it seems to just define and assign a bunch of local variables